src/provider/ClickstreamProvider.ts

```diff
@@ -227,13 +225,6 @@ export class ClickstreamProvider implements AnalyticsProvider {
 		this.recordEvent(errorEvent);
 	}
 
-	recordProfileSet() {
```

Comment:
Just see removing the profileSet(), why naming the pr as feature?

Reply:
this pr generated from cherry-pick this [commit](https://github.com/awslabs/clickstream-web/pull/22/commits/24c08f825ce9cec849dc9615d9a5e88d579b12e5) into main branch, The strange thing is that this commit appears in the previous PR. Maybe I got confused by switching between the SDKs. 

I'm thinking about whether to remove the previous commit and resubmit it, or directly modify the message content of the previous PR submission to include this feature.